kubernetes/echo.yaml

```diff
@@ -27,7 +27,7 @@ spec:
     app: esp-echo
   type: LoadBalancer
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: esp-echo
```

Comment:
Hi @nareddyt do we need to add following lines

 selector:
    matchLabels:
      app: esp-echo

Reply:
Yes, `selector` is required when using `apps/v1`. @TAOXUY did this deploy work for you? Maybe it works on kubernetes < 1.16, but not on 1.16

Reply:
Done. Verified in cluster version 1.16.